examples/reload-persistence-unit-properties/README_pt.adoc

```diff
@@ -0,0 +1,57 @@
+:index-group: JPA
+:jbake-type: page
+:jbake-status: status=published
+= Recarregar propriedades do persistence unit
+
+Este exemplo tem como objetivo simular uma campanha de benchmark no JPA.
+
+Primeiro, você executará seu aplicativo e perceberá que precisa de um L2
+cache para estar em conformidade com o SLA (Service Level Agreement).
+
+Então, você altera sua configuração persistence.xml e reinicia seu
+aplicação, você espera um pouco porque está usando o OpenEJB;)… mas você
+espera.
+
+Portanto, para tentar ir mais rápido em uma campanha longa, basta alterar sua configuração em tempo de execução para testá-la e, quando funcionar, alterar seu arquivo de configuração para manter a modificação.
+
+Para fazer isso, podemos simplesmente usar o JMX.
+
+O OpenEJB registra automaticamente um MBean por entitymanager (persistence unit).
+
+Ele permite que você altere as propriedades do persistence unit, mesmo que mais seja possível.
+
+== Testando
+
+O teste é simples: persistimos uma entidade, a consultamos três vezes
+sem cache, então ativamos o cache e realizamos a execução novamente
+consulta que um é suficiente para fazer o mesmo.
```

Comment:
This sentence is confusing. Even in English it is not very clear.

Reply:
I will re-read and see how I can improve.

thank you.